scripts/variantstore/wdl/GvsUtils.wdl

```diff
@@ -384,14 +384,18 @@ task BuildGATKJar {
   }
 }
 
-task CreateDataset {
+task CreateDatasetForTest {
   input {
     String? git_branch_or_tag
     String dataset_prefix
     String dataset_suffix
     String cloud_sdk_docker
+    # By default auto-expire tables 2 weeks after their creation. Unfortunately there doesn't seem to be an automated way
+    # of auto-expiring the dataset, but the date is in the dataset name so old datasets should be easy to identify.
+    Int? table_ttl_seconds = 2 * 7 * 24 * 60 * 60
   }
   meta {
+    description: "Create a dataset for testing purposes whose tables are all set to auto-expire. Do not call this task for production code as the tables created within it will auto-delete!"
```

Comment:
Is it the tables or the entire dataset that auto-expires? (I honestly don't know, but assume the former).

Reply:
It's the tables that auto-expire, I didn't see a way to auto-expire whole datasets unfortunately.